code/game/objects/items/weapons/teleportation.dm

```diff
@@ -127,7 +127,7 @@ Frequency:
 		if(T.x>world.maxx-8 || T.x<8)	continue	//putting them at the edge is dumb
 		if(T.y>world.maxy-8 || T.y<8)	continue
 		A = get_area(T)
-		if(A.tele_proof == 1) continue // Telescience-proofed areas require a beacon.
+		if(A.tele_proof) continue // Telescience-proofed areas require a beacon.
```

Comment:
would you make this indent properly?

Reply:
I'm unsure what you are refering to? Do you mean indent the continue to line up with the ones above, or the comment, or both?

Reply:
Finally figured out what you meant by this ;p